operations/mimir/rollout-operator.libsonnet

```diff
@@ -53,9 +53,14 @@
   rollout_operator_container::
     container.new('rollout-operator', $._images.rollout_operator) +
     container.withArgsMixin($.util.mapToFlags($.rollout_operator_args)) +
-    container.withPorts([
-      $.core.v1.containerPort.new('http-metrics', 8001),
-    ]) +
+    container.withPorts(
```

Comment:
This change makes sense to me but can be rewritten to something like:

```
container.withPorts(
  [$.core.v1.containerPort.new('http-metrics', 8001)] + if $._config.enable_rollout_operator_webhook then [$.core.v1.containerPort.new('https', 8443)] else []
)
```

Reply:
I agree it looks better.